src/sentry/integrations/gitlab/client.py

```diff
@@ -134,19 +135,22 @@ def get_user(self):
         """
         return self.get(GitLabApiClientPath.user)
 
-    def search_group_projects(self, group, query=None, simple=True):
-        """Get projects for a group
+    def search_projects(self, group=None, query=None, simple=True):
+        """Get projects
 
         See https://docs.gitlab.com/ee/api/groups.html#list-a-group-s-projects
+        and https://docs.gitlab.com/ee/api/projects.html#list-all-projects
         """
 
         def gen_params(page_number, page_size):
             # simple param returns limited fields for the project.
             # Really useful, because we often don't need most of the project information
+            if group:
             return {
-                "search": query,
-                "simple": simple,
-                "include_subgroups": self.metadata.get("include_subgroups", False),
+                    "search": query,
+                    "simple": simple,
```

Comment:
This is indented one too many tabs and causing the linting error in CI

Reply:
I went through it, it got lost in rebase or sth. I tried to review the intention and adapt the code to the current state of master